Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml.cs

```diff
@@ -17,11 +18,11 @@ public class SupplierVatQuestionModel(IOrganisationClient organisationClient,
     const string VatSchemeName = "VAT";
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```

Comment:
Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors